_pkgdown.yml

```diff
@@ -250,6 +251,11 @@ reference:
   - temp.options
   - which
   - with-context-manager
+  - hasFunction
```

Comment:
Nah, give this `#' @keywords internal` and don't include it in the reference index.